src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -25,6 +25,42 @@ class UserInfoSection extends React.Component {
       );
     }
 
+    let entitlementInfo;
+    if (enrollmentData.percentageBenefit === 0 && enrollmentData.originalEntitlement === 0) {
```

Comment:
It looks like `currentlyAllowed` is defined as both of these values being non-zero, and currently disallowed is defined as both of them being zero, so (while I'm guessing this never happens), the case when one is zero and one is non-zero falls between the cracks. I say we just treat that case as allowed (if one is nonzero, you're probably getting benefits), so above we would set
```
const currentlyAllowed = enrollmentData.percentageBenefit !== 0 || enrollmentData.originalEntitlement !== 0;
```
and here just use `!currentlyAllowed`